docs/shader-modules/project.md

```diff
@@ -137,6 +137,15 @@ vec4 project_common_position_to_clipspace(vec4 position)
 Converts the coordinates of a point from the common space to the clip space, which can be assigned to `gl_Position` as the "return value" from the vertex shader.
 
 
+### project_get_orientation_matrix
+
+```glsl
+mat3 project_get_orientation_matrix(vec3 up)
```

Comment:
This up vector is also defined in common space? Or is it a lng/lat thing? Can we specify specify?